frontend/src/components/worksheets/BundleDetail/SideBar.jsx

```diff
@@ -65,82 +62,181 @@ class SideBar extends React.Component<
-                    uuid={ bundleInfo.uuid }
-                    value={ metadata.name }
-                    canEdit={ hasEditPermission && editableMetadataFields.includes("name") }
-                    onChange={ (name) => onUpdate({ name }) }
-                />
+                <div className={classes.wrappableText}>
+                    <BundleEditableField
+                        dataType={ metadataType.name }
+                        fieldName="name"
+                        uuid={ bundleInfo.uuid }
+                        value={ metadata.name }
+                        canEdit={ hasEditPermission && editableMetadataFields.includes("name") }
+                        onChange={ (name) => onUpdate({ name }) }
+                    />
+                </div>
                 {/** ----------------------------------------------------------------------------------------------- */}
                 <ConfigLabel label="Description" />
-                <BundleEditableField
-                    dataType={ metadataType.description }
-                    fieldName="description"
-                    uuid={ bundleInfo.uuid }
-                    value={ metadata.description }
-                    canEdit={ hasEditPermission && editableMetadataFields.includes("description") }
-                    onChange={ (description) => onUpdate({ description }) }
-                />
+                <div className={classes.wrappableText}>
+                    <BundleEditableField
+                        dataType={ metadataType.description }
+                        fieldName="description"
+                        uuid={ bundleInfo.uuid }
+                        value={ metadata.description }
+                        canEdit={ hasEditPermission && editableMetadataFields.includes("description") }
+                        onChange={ (description) => onUpdate({ description }) }
+                    />
+                </div>
                 {/** ----------------------------------------------------------------------------------------------- */}
-                { isRunBundle && <Dependency bundleInfo={ bundleInfo }/> }
-                { metadata.data_size && <ConfigLabel label="Size" /> }
-                { metadata.data_size && renderFormat(metadata.data_size, metadataType.data_size) }
-                <ConfigLabel label="Created On" />
-                {renderFormat(metadata.created, metadataType.created)}
-                <ConfigLabel label="Owner" />
-                { bundleInfo.owner.user_name }
-                <ConfigLabel label="Attached to Worksheets" />
-                {
-                    bundleInfo.host_worksheets.map((worksheet) =>
-                        <div
-                            key={ worksheet.uuid }
+                <div>
+                    <ConfigLabel label="Status" inline={true}/>
+                    <div className={ `${ classes.stateBox } ${ classes[stateSpecClass] }`} style={{ display: 'inline' }}>

```

Comment:
Slight alignment issue here, could you fix?

![image](https://user-images.githubusercontent.com/1689183/67430649-8b73ce00-f597-11e9-8b85-3c04897bcae2.png)